docs/src/main/sphinx/sql/set-path.rst

```diff
@@ -0,0 +1,54 @@
+========
+SET PATH
+========
+
+Synopsis
+--------
+
+.. code-block:: text
+
+    SET PATH path-element[, ...]
+
+Description
+-----------
+
+Define a collection of paths to functions or table functions in specific
+catalogs and schemas for the current session.
+
+Each path-element uses a period-separated syntax of to specify the catalog name
+and schema location ``<catalog>.<schema>``of the function or only the schema
+location ``<schema>`` in the current catalog. The current catalog is set with
+:doc:`use` or as part of a client tool connection. Catalog and schema must
+exist.
+
+Function names must be unique across all path elements.
+
+Examples
+--------
+
+The following example sets a path to access functions in the ``system`` schema
```

Comment:
It might be good to mention that no matter what path you set, `global.builtin` is always tried first.

Reply:
Add that in the description section